README.md

```diff
@@ -34,6 +36,32 @@ As the enhancement proposal reveals, this is not a fully compliant CSI Driver im
 solely provides the minimum amounts of the Kubernetes / CSI contract needed to achieve the goals stipulated in the 
 Enhancement proposal.
 
+## Current status with respect to the Kubernetes CSIVolumeSource API
+
+So let's take each part of the [CSIVolumeSource struct](https://github.com/kubernetes/api/blob/71efbb18d63cd30604981514ac623a6be1d413bb/core/v1/types.go#L1743-L1771):
+
+- for the `Driver` string field, it needs to be ["csi-driver-projected-resource.openshift.io"](https://github.com/openshift/csi-driver-projected-resource/blob/1fcc354faa31f624086265ea2228661a0fc2e7b1/pkg/client/client.go#L28)
+- for the `VolumeAttributes` map, this driver currently adds the "share" key (which maps the the `Share` instance your `Pod` wants to use) in addition to the 
+elements of the `Pod` the kubelet stores when contacting the driver to provision the `Volume`.  See [this list](https://github.com/openshift/csi-driver-projected-resource/blob/c3f1c454f92203f4b406dabe8dd460782cac1d03/pkg/hostpath/nodeserver.go#L37-L42)
+- the `ReadOnly` field is ignored, as the this driver's controller actively updates the `Volume` as the underlying `Secret` or `ConfigMap` change.
+- the `FSType` field is ignored.  This driver by design only supports `tmpfs`, with a different mount performed for each `Volume`, in order to defer all SELinux concerns to the kubelet.
+- the `NodePublishSecretRef` field is ignored.  The CSI `NodePublishVolume` and `NodeUnpublishVolume` flows gate the permission evaluation required for the `Volume`
+by performing `SubjectAccessReviews` against the reference `Share` instance, using the `serviceAccount` of the `Pod` as the subject.
+  
+## Current status with respect to the Kubernetes VolumeMount API
+
+So let's take each part of the [VolumeMount struct](https://github.com/openshift/csi-driver-projected-resource/blob/c3f1c454f92203f4b406dabe8dd460782cac1d03/pkg/hostpath/nodeserver.go#L37-L42),
+where each `Container` in a `Pod` is allowed an array of such mounts:
+
+- the validations that Kubernetes applies to `Name` and `MountPath` fields apply
+- if the `MountPath` of one `VolumeMount` is a subdirectory of another `MountPath` in the `Container`, Kubernetes will allow it, but this driver 
+  currently does not support that scenario, though may soon
+- the `ReadOnly` field is currently ignored, per our explanation above with [CSIVolumeStruct](#current-status-with-respect-to-the-kubernetes-csivolumesource-api)
```

Comment:
Isn't it `CSIVolumeSource` struct, not `CSIVolumeStruct` ?

Reply:
sure is ... will update in next push